pkg/common/parameters.go

```diff
@@ -42,7 +42,7 @@ const (
 
 // DiskParameters contains normalized and defaulted disk parameters
 type DiskParameters struct {
-	// Values: pd-standard OR pd-ssd
+	// Values: PD disk type (see GCP documentation. This driver does not validate this field).
```

Comment:
nit: keeping it consistent with comments on other fields, we can put the actual values. ("pd-standard","pd-ssd", "pd-balanced")

Reply:
Done.